### ISSUE TITLE
AssetManager could not be initialized

### ISSUE THREAD

**Issue by Avatar4eg** [zfcampus/zf-apigility#172]
After clean installation:
1. composer create-project -sdev zfcampus/zf-apigility-skeleton
2. composer update
I have an error:
`Zend\ModuleManager\Exception\RuntimeException: Module (AssetManager) could not be initialized. in C:\_work\projects\sites\site_dotaquiz\vendor\zendframework\zend-modulemanager\src\ModuleManager.php on line 203`

Tried to install [zfcampus/zf-asset-manager](https://github.com/zfcampus/zf-asset-manager).

**Comment by webimpress** [comment]
@Avatar4eg `zfcampus/zf-apigility-skeleton` is not yet updated to use ZF3 modules, please use `zf-apigility` v1.2

**Comment by Avatar4eg** [comment]
I also use [zfcampus/zf-apigility-doctrine](zfcampus/zf-apigility-doctrine), which has version conflicts if packages not updated.

**Comment by webimpress** [comment]
@Avatar4eg  Yes, I'm working on it, doctrine modules are not yet release to support ZF3 modules, need to be patient to use it will ZF3. I'd suggest to use it on ZF2 for now or wait few more days and start on ZF3.

**Comment by jamjan** [comment]
In my case what helped is  `composer require rwoverdijk/assetmanager`

**Comment by webimpress** [comment]
@jamjan It is possible now because `rwoverdijk/assetmanager` is finally updated to support ZF2 and ZF3. More about this change in apigility you can read here:
https://apigility.org/documentation/intro/updating-to-1-4#asset-management

**Comment by weierophinney** [comment]
This repository has been closed and moved to laminas-api-tools/api-tools; a new issue has been opened at https://github.com/laminas-api-tools/api-tools/issues/21.
